server/src/main/scala/drt/chroma/ArrivalsDiffingStage.scala

```diff
@@ -26,19 +27,24 @@ final class ArrivalsDiffingStage(initialKnownArrivals: Seq[Arrival]) extends Gra
 
     setHandlers(in, out, new InHandler with OutHandler {
       override def onPush(): Unit = {
+        val start = SDate.now()
         if (maybeResponseToPush.isEmpty) {
           log.info(s"Incoming ArrivalsFeedResponse")
           maybeResponseToPush = processFeedResponse(grab(in))
         } else log.info(s"Not ready to grab until we've pushed")
 
         if (isAvailable(out))
           pushAndClear()
+
+        log.info(s"onPush Took ${SDate.now().millisSinceEpoch - start.millisSinceEpoch}ms")
```

Comment:
maybe indicate somehow that this is something we will want to move to an analytics type platform - or make it debug.